_docs-sources/guides/stay-up-to-date/patcher/index.md

```diff
@@ -0,0 +1,78 @@
+# Patcher
+
+## Installation
+
+To install Patcher, download the appropriate package for your system from the [releases](https://github.com/gruntwork-io/patcher-cli/releases) page.
+
+### For MacOS and Linux users
+
+Patcher runs as a single binary called `patcher`. Make sure it's available within your `PATH`.
+
+1. After downloading Patcher, move it to your desired destination directory. If you're using MacOS, you will have to unzip the package. For other operating systems, the artifact you download is the executable itself.
+2. If necessary, rename the binary to `patcher`.
+```bash
+mv patcher_linux_amd64 patcher
+```
+3. Move `patcher` to a location available in your `PATH`.
+```bash
+echo $PATH
```

Comment:
For examples where the second command is dependent on the output of the first, I find it useful to show the output of the first command. e.g.,

```suggestion
echo $PATH
> /usr/local/bin
mv patcher /usr/local/bin
```

Reply:
I will add an example of full path, hopefully this avoids confusion.